python/sdk/merlin/model.py

```diff
@@ -619,6 +619,17 @@ def set_traffic(self, traffic_rule: Dict['ModelVersion', int]) \
 
         return ModelEndpoint(ep)
 
+    def delete_model(self) -> int:
+        """
+        Delete this model. Please note that any inactive related entity (model versions, endpoints and prediction jobs) will get deleted by this process.
+        Deletion will failed if there are any active related entity (model versions, endpoints and prediction jobs)
```

Comment:
Nit: Here and in other instances, `Deletion will failed` -> `Deletion will fail`